src/main/java/dev/jbang/cli/Init.java

```diff
@@ -179,10 +179,11 @@ String getRenderedScriptOrFile(Map<String, String> fileRefs, List<RefTarget> ref
 			Optional<RefTarget> optionalRefTarget = refTargets	.stream()
 																.filter(refTarget -> refTarget	.getSource()
 																								.getFile()
```

Comment:
Don't use `getFile()` here because that can be a cached/rewritten file that doesn't follow the original path and file names. That why the `endWith()` didn't work. Replace with 

```
.getSource()																						.getOriginalResource()																						.endsWith(...)
```

and things should work fine.

Reply:
done. Works as expected. JUnit templates use now get correct output.